### DISCUSSION TITLE
Give better explanation of difference from Xournal

### DISCUSSION THREAD

**Discussion by eyalroz** [xournalpp/xournalpp#2594] [Q&A]
I've been an avid user of Xournal (not Xournal++, which I've just noticed) for several years now, as it is available as a package on Debian-based systems. 

Although I haven't mastered it - I am used to it, with its UI quirks. It also works stably for me.

Now I see this new shiny rewrite, and I'm wondering: "Should I try it? What will I get out of it?"

I believe that at the very least the FAQ should include a listing of:

* Features available in Xournal++ but not in Xournal
* Features available in Xournal but not in Xournal++ (if there are any)
* Known issues/bugs/inconveniences of Xournal, resolved in Xournal++
* Stability benefits of Xournal over Xournal++, if there are any
* Performance/speed/resource consumption comparison of Xournal vs Xournal++, if relevant

Other than that - thank you all for your efforts.

PS - This issue should have a "documentation" tag, but "enhancement" was the closest thing I could choose.

**Comment by rolandlo** [comment]
This may have its proper place on the [work-in-progress website](https://xournalpp.github.io/). If you agree, the best place to report this would be [this issue](https://github.com/xournalpp/xournalpp.github.io/issues/21), where we discuss what content to add to the website.

**Comment by eyalroz** [comment]
Well, there too, perhaps - but this would also fit in the FAQ here. (Or maybe you're going to link the FAQ here to the website instead?)

Anyway, if you add it to that website, that would certainly be nice.

**Comment by rolandlo** [comment]
Yes, I think we'd rather link the FAQ here to the website. An average user would probably not look for the FAQ at the github repo, but rather look at the website FAQ section (if we create one).

**Comment by rolandlo** [comment]
@eyalroz I have created a first draft for the comparison of Xournal++ with Xournal for the website, see [this pull request](https://github.com/xournalpp/xournalpp.github.io/pull/23/files). While it is still incomplete, it will give you some information that you may have known yet.

**Comment by eyalroz** [comment]
@rolandlo : Oh, that's great! It's basically what I wanted to know.

**Comment by pedro-nonfree** [comment]
which is by the way exactly to this page right now https://xournalpp.github.io/community/other-software/#xournal
